Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:carbon-alpine
```

Comment:
You might want to swap this to `node:alpine` so you can use `npm audit`, or do you want to wait for a named LTS Docker image?